modules/Hlsjs/resources/hlsjs.js

```diff
@@ -331,20 +331,19 @@
 				var audioTracks = this.hls.audioTracks;
 				if (audioTracks && audioTracks.length > 0) {
 					var audioTrackData = {languages: []};
-					var audioTrackLangs = {};
+					var createAudioTrack = function(audioTrack) {
+						return {
+							'kind': 'audioTrack',
+							'language': audioTrack.lang,
+							'srclang': audioTrack.lang,
+							'label': audioTrack.name,
+							'title': audioTrack.name,
+							'id': audioTrack.id,
+							'index': audioTrackData.languages.length
+						};
+					};
 					$.each(audioTracks, function (index, audioTrack) {
-						if (audioTrackLangs[audioTrack.lang] === undefined) {
-							audioTrackLangs[audioTrack.lang] = 1;
```

Comment:
What's about duplicate audios?

Reply:
As I wrote, it was never an issue in hls, just dash that we had multi bitrate(representation) per one audio adaptation, this doesn't exist at all in hls.

Reply:
As I remember we had 2 'undefined' audio labels. this scenario cannot be with hls?